release(core): prepare release notes (#404)
centreon/centreon-documentation: en/releases/centreon-core.md

## Centreon Web release notes

### 20.10.0

### Enhancements

- [API/Core/Configuration] Log actions when adding/enabling/disabling hosts
- [API] Possibility to Register servers (Remote Server, Poller, Centreon Map)
- [Configuration/Wizard] Add possibility to select registered poller
- [Authentication] Replace Keycloak to generic OAuth2 / OpenId Connect
- [Event Logs] Display anomaly detection as regular service
- [Monitoring/Resources Status] Add shortcuts for hosts and services details
- [Monitoring/Resources Status] Add timeline for hosts and services details

> - [Monitoring/Resources Status] Be alble to filter on status output

Reviewer: ```suggestion
- [Monitoring/Resources Status] Be able to filter on status output
```

- [Monitoring/Resources Status] Add possibility to save/manage filters
- [Monitoring/Resources Status] Add possibility to submit result for resources
- [Monitoring/Resources Status] Redirect all realtime links to Resources Status page
- [Remote Server] Replace HTTP flow by gorgone between Central and Remote Servers

## Centreon Engine release notes

### 20.10.0

> Known behaviours:
>
>   - If Engine on a Poller or Remote Server is not upgraded to 20.10,
>     configuration files copied from an upgraded Central by Gorgone using ZMQ
>     communication will not be readable by Engine process, resulting to this
>     error in Engine log:
>
>     ```text
>     Error: Parsing of global configuration failed: Can't open file '/etc/centreon-engine/centengine.cfg'
>     ```
>

> >     This is due to more strict rights on those files.

Reviewer 2: ```suggestion
>     This is due to stricter rights on those files.
```

>

> >     As always, we **strongly recommand** to upgrade all components to

Reviewer 2: ```suggestion
>     As always, we **strongly recommend** to upgrade all components to
```

> >     As always, we **strongly recommand** to upgrade all components to

Reviewer 2: ```suggestion
>     As always, we **strongly recommend** to upgrade all components to
```

> >     match Central serie.

Reviewer 2: ```suggestion
>     match the Central server's version.
```

> >     match Central serie.

Reviewer 2: ```suggestion
>     match the Central  server's version.
```

>
>     However, during an upgrade process, it is possible to manually

> >     add the user *centreon-engine* to *centreon-gorgone* group with

Reviewer 2: ```suggestion
>     add the user *centreon-engine* to the *centreon-gorgone* group with
```

> >     add the user *centreon-engine* to *centreon-gorgone* group with

Reviewer 2: ```suggestion
>     add the user *centreon-engine* to the *centreon-gorgone* group with
```

>     the following command:
>
>     ```shell
>     usermod -a -G centreon-gorgone centreon-engine
>     ```

### Bugfixes

- Contains all fixes up to version 20.04.7

## Centreon Broker release notes

### 20.10.0

> Known behaviours:
>
>   - If Broker on a Poller or Remote Server is not upgraded to 20.10

> >     (or with a version prior to 20.04.9), the communication between those

Reviewer 2: ```suggestion
>     (or with a version prior to 20.04.9), the communication between said
```

> >     (or with a version prior to 20.04.9), the communication between those

Reviewer 2: ```suggestion
>     (or with a version prior to 20.04.9), the communication between said
```

>     Poller or Remote Server and an upgraded Central may not work.
>

> >     As always, we **strongly recommand** to upgrade all components to match

Reviewer 2: ```suggestion
>     As always, we **strongly recommend** to upgrade all components to match
```

> >     As always, we **strongly recommand** to upgrade all components to match

Reviewer 2: ```suggestion
>     As always, we **strongly recommend** to upgrade all components to match
```

> >     Central serie.

Reviewer 2: ```suggestion
>     the Central server's version.
```

> >     Central serie.

Reviewer 2: ```suggestion
>     the Central server's version.
```

>
>     However, during an upgrade process, communication can be maintained

> >     by making sure Brokers' configurations match the following conditions:

Reviewer 2: ```suggestion
>     by making sure Broker's configurations match the following conditions:
```

> >     by making sure Brokers' configurations match the following conditions:

Reviewer 2: ```suggestion
>     by making sure Broker's configurations match the following conditions:
```
